test/removeNPMAbsolutePaths.js

```diff
@@ -149,6 +149,28 @@ describe('removeNPMAbsolutePaths.js', function () {
           });
       });
 
+      it('rewrite pacakge.json, but only specific fields', function () {
+        const dirPath = `${__dirname}/data/specific_fields`;
+        const filePath = `${dirPath}/module/package.json`;
+        const opts = {
+          userSpecifiedKeys: true,
+          keys: ['_inBundle', '_where'],
+        };
+        const promise = removeNPMAbsolutePaths(dirPath, opts);
+        return expect(promise).be.fulfilled
+          .then((results) => {
+            expect(results).to.be.an('array').that.have.lengthOf(3);
+            const fileResults = results.find(result => result.filePath === filePath);
+            expect(fileResults).to.include({ success: true, rewritten: true });
```

Comment:
To make this more reliable you probably want to check that `_inBundle` and `_where` are present in the results but, for example, `_from` is not.

Reply:
am I missing an example where the tests expect certain content written out?  It looked like from the stubbed `writeFile` that nothing was actually being saved to the file system.